portfolio/src/main/webapp/script.js

```diff
@@ -37,11 +37,11 @@ function addRandomQuote() {
  * This function opens the tab we want to access and hides the unnecessary tabs
  */
 function openTab(evt, tabName) {
-    var tabcontent = document.getElementsByClassName("tabcontent");
+    const tabcontent = document.getElementsByClassName("tabcontent");
     for(var i = 0; i <tabcontent.length; i++) {
         tabcontent[i].style.display = "none";
     }
-    var tablinks = document.getElementsByClassName("tablinks");
+    const tablinks = document.getElementsByClassName("tablinks");
     for(i = 0; i< tablinks.length; i++) {
```

Comment:
This would be using the var from before?

Reply:
Not 100% clear on what you're asking here: are you wondering about the difference between tablinks and tabcontent & when I'm using them? Or the variable declaration?

Reply:
local variable i isn't declared here so I wasn't sure if it was working/why its working

Reply:
Whoops. Thanks for catching that!